### PR TITLE
Updates to CD docs

### PR DESCRIPTION
**Pull request by JeffNaef-Triggermesh**

### CODE REVIEW

File: docs/guides/cd.md [hunk 1]
```diff
@@ -1,64 +1,64 @@
...
-**Begin by forking the [sample repository](https://github.com/sebgoa/tmserverless) which contains a function and an example `serverless.yaml` manifest.**
+###### The following example uses this [sample repository.](https://github.com/sebgoa/tmserverless) To follow along begin by forking the example to your repository provider. 
 
-Select the _Repositories_ section from the Navigation menu and then _CREATE NEW_ 
+1. Select the _Repositories_ section from the Navigation menu and then _CREATE NEW_ 
 
 ![](../images/tmRepoView.png)
 
-Select a source control provider. Upon submission of this form you will be asked to authenticate with the providor chosen in the previous step.
+2. Select a source control provider. Upon submission of this form you will be asked to authenticate with the chosen provider.
```
Comment (cab105) [inline docs/guides/cd.md:16]: A neat trick with markdown is that you can just restate `1.` and it should automagically do the right thing with ordering.

File: docs/guides/cd.md [hunk 2]
```diff
@@ -1,64 +1,64 @@
...
-The **_Repositories_** section in the console contains a view to register any repository, from previously linked accounts, for continuous deployment.
 
+The Triggermesh console provides an interface called [Repositories.](https://cloud.triggermesh.io/function-triggers) Here the user is able to select a codebase, from a linked repository provider, to be continuously deployed. This codebase **must** contain a deployment manifest[^1] . 
 
-## Register the repository
 
+## Register the repository
 
-**Begin by forking the [sample repository](https://github.com/sebgoa/tmserverless) which contains a function and an example `serverless.yaml` manifest.**
+###### The following example uses this [sample repository.](https://github.com/sebgoa/tmserverless) To follow along begin by forking the example to your repository provider.
```
Comment (cab105) [inline docs/guides/cd.md:10]: Why go with the sub-level vs keeping the bold typesetting?

File: docs/guides/cd.md [hunk 3]
```diff
@@ -1,64 +1,64 @@
...
 
-Once registration is complete the _Repositories_ view will contain your function project. You can disable the registration at any time by selecting the project and clicking on the _Disable Selected_ button.
+5. Once registration is complete the _Repositories_ view will contain your function project. You can disable the registration at any time by selecting the project and clicking on the _Trash Can_ icon located next to _REFRESH_
 
 ![](../images/repowiz4.png)
 
 ## Viewing The Deployment
 
-After every sucessful registration of a new repository a _Task_ will be created for it.
+###### After every sucessful registration of a new repository a _Task_ will be created for it.
```
Comment (cab105) [inline docs/guides/cd.md:35]: Why the sublevel here?  Perhaps something to emphasize the text?

File: docs/guides/cd.md (hunk 1 above)
Comment (JeffNaef-Triggermesh) [reply docs/guides/cd.md:16]: oh cool! thank you!

File: docs/guides/cd.md (hunk 3 above)
Comment (JeffNaef-Triggermesh) [reply docs/guides/cd.md:35]: This one I agree should be put back to bold.

File: docs/guides/cd.md (hunk 2 above)
Comment (JeffNaef-Triggermesh) [reply docs/guides/cd.md:10]: This is more of a note or an aside then a statement. I wanted it to be small and its appearance to reflect its intention as a side note. If you think we should move it back to bold or regular i have no issues

Comment (cab105) [reply docs/guides/cd.md:10]: Fair enough.  Although addressed on slack, for others who may see this later, the heading indicator could be parsed to automatically generate things like a table of contents and would not be a good use here.  My personal convention is use italics for notes and bold for important items.

**Comment by sebgoa** [comment]
is this ready for merge ?

**Comment by cab105** [comment]
@sebgoa I'm still reviewing this.

**Pull request merged**